styles/styles.css

```diff
@@ -326,3 +326,31 @@ main .cyan-headings {
 main .blue-headings {
   --color-heading: var(--color-blue);
 }
+
+/* blog styles */
+
+.blog-page div.related-content {
+  background-color: #f2f6f7;
+  width: 100%;
+}
+
+@media (width >= 992px) {
```

Comment:
I believe we have decided to not use this style of media query due to browser compat issues with older versions of iOS Safari - see https://github.com/aemsites/stericycle-shared/pull/33/files

Reply:
We added a stylelint rule to check for this as part of that PR so we should be good moving forward.

Reply:
@seanathero fixed. I also added the stylelint fix to this branch